ftl/core/adding.ftl

```diff
@@ -6,4 +6,6 @@ adding-history = History
 adding-note-deleted = (Note deleted)
 adding-shortcut = Shortcut: { $val }
 adding-the-first-field-is-empty = The first field is empty.
-adding-you-have-a-cloze-deletion-note = You have a cloze deletion note type but have not made any cloze deletions. Proceed?
+adding-missing-cloze = You have a cloze notetype but have not made any cloze deletions.
```

Comment:
This should probably be a warning, not an error. Some users want to quickly dump some text into Anki and then go back and edit it later, and they now how no way to proceed.

Reply:
(I seem to recall Anki originally had this as a hard error, and some people pushed back on it, resulting in it being turned into a warning)